superset/connectors/sqla/models.py

```diff
@@ -649,7 +649,7 @@ def get_sqla_query(  # sqla
             elif m in metrics_dict:
                 metrics_exprs.append(metrics_dict.get(m).get_sqla_col())
             else:
-                raise Exception(_("Metric '{}' is not valid".format(m)))
+                raise Exception(_(f"Metric '{m}' is not valid"))
```

Comment:
Ditto re the error message here? `Metric '{m}' does not exist` / `... not found`?